src/html.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { ReactNode } from 'react';
+import cheerio from 'cheerio';
+import { Stream } from 'stream';
+/**
+ *
+ * Gatsby does a two-pass render for HTML. It loops through your pages first
+ * rendering only the body and then takes the result body HTML string and
+ * passes it as the `body` prop to this component to complete the render.
+ */
+
+const postProcess = (body: string) => {
+  let globalStyles = '';
+
+  if (body.includes('<template')) {
+    const $ = cheerio.load(body);
+    const templates = $('template');
+
+    templates.toArray().forEach(element => {
+      const str = $(element).html();
+      const styles = cheerio.load(String(str))('style');
```

Comment:
Why is `String` needed?
```suggestion
      const styles = cheerio.load(str)('style');
```

Reply:
Casting to make ts happy, str can be null.